frontend/src/v5/ui/themes/global.ts

```diff
@@ -25,16 +26,40 @@ export const GlobalStyle = createGlobalStyle`
 		position: relative;
 		overflow-y: hidden;
 	}
-	
+
 	body {
 		margin: 0;
 		padding: 0;
 		${({ theme }: { theme: Theme }) => theme.typography.body1};
 	}
-	
+
 	#app {
 		height: 100%;
 		display: flex;
 		flex-direction: column;
 	}
+
+	#viewer #unityViewer {
+		position : absolute;
+		height: 100% !important;
+		width: 100% !important;
+		overflow: hidden;
+		margin-top: 72px;
+		background: ${COLOR.TRANSPARENT} !important;
+	}
+
+	.emscripten {
+		background: ${COLOR.TRANSPARENT} !important;
+	}
+
+	div#unityViewer.emscripten canvas {
+		background: ${COLOR.TRANSPARENT} !important;
+	}
```

Comment:
this and the selector above apply the same rules, would it be possible to join them?`
	.emscripten,
	div#unityViewer.emscripten canvas {
		background: ${COLOR.TRANSPARENT} !important;
	}
`